logic/src/main/java/org/itsallcode/whiterabbit/logic/service/AppService.java

```diff
@@ -20,7 +21,7 @@ public AppService(Storage storage, ClockService clock) {
 		this.clock = clock;
 	}
 
-	public void update() {
+	public DayRecord update() {
 		final LocalDate today = clock.getCurrentDate();
 		final MonthIndex month = storage.loadMonth(today);
 		final DayRecord day = month.getDay(today);
```

Comment:
Below: don't use null checks outside of the day class introduce proper boolean methods for this kind of check.